### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by jjasghar**
Updated so the output is now done in cards

- Added help
- Added card for nodes
- Added card for environments
- Added unikitty prettiness

### CODE REVIEW

**Comment by rmanalan** [comment]
LGTM

File: routes/index.js [hunk 1]
```diff
@@ -324,6 +418,31 @@ module.exports = function (app, addon) {
             }
            );
 
+
```
Comment (dosman711) [inline routes/index.js]: You shouldn't check in commented code :)

File: routes/index.js [hunk 2]
```diff
@@ -290,9 +346,47 @@ module.exports = function (app, addon) {
...
+                         hipchat.sendMessage(req.clientInfo, req.context.item.room.id, msg, options, card)
+                             .then(function (data) {
+                                 res.send(200);
+                             });
+
                      } else {
 
-                         stringResult = "chef command not recognized.";
+                         stringResult = "chef command not recognized. Try /chef help";
```
Comment (dosman711) [inline routes/index.js]: A nit, but consider `recognized, try '/chef help'`

File: routes/index.js [hunk 3]
```diff
@@ -165,17 +165,39 @@ module.exports = function (app, addon) {
...
+                                 "style": "application",
+                                 "url": "https://" + jsonResult.fqdn,
+                                 "id": uuid.v4(),
+                                 "title": jsonResult.fqdn,
+                                 "description": fqdnString + " successfully converged " + timeString + " ago.",
+                                 "icon": {
+                                     "url": "http://emojipedia-us.s3.amazonaws.com/cache/1b/df/1bdf10d829ad6873c0f5157b263178ec.png"
+                                 }
+                             };
+                             var msg = '<b>' + card.title + '</b>: ' + card.description;
```
Comment (dosman711) [inline routes/index.js]: Pick either `'` or `"` and change all of the quotes to that style. IIRC the standard JS style doc recommends single quotes.

**Pull request merged**